### DISCUSSION TITLE
Issues in Methods Geolocator

### DISCUSSION THREAD

**Discussion by celsoabreu** [flet-dev/flet#3562] [General]
import flet as ft
from flet import * 

async def main(page: Page):
    #page.window.always_on_top = True
    page.on_error = lambda e: print(f"Page Error: {e.data}")
    page.scroll = ScrollMode.ADAPTIVE
    page.appbar = AppBar(title=ft.Text("Geolocator Tests"))
    geolocator = Geolocator()
    page.overlay.append(geolocator)
    Lok = geolocator.is_location_service_enabled()
    print(geolocator, geolocator, type(geolocator)) 
    
    page.add(Text(value="Hello, world!"), 
         Text(f"Geolocator: {geolocator}"),
         Text(f"Type: {type(geolocator)}"), 
    )

    #page.add(Text('Gelocalização do usuario', weight='bold', title_size=15,))
        
    if Lok == False:
        page.apennd(msg = Text("Voce precisa ativar a localização do seu dispositivo location/app settings. "))
    else:
        local = geolocator.get_current_position()
        print(local, {local.latitude}, {local.longitude})

        if local == False:
            page.apennd(msg = Text("Voce precisa ativar a localização do seu dispositivo location/app settings. "))
            page.update()
        else:
            page.apennd(Text(f"get_current_position: ({local.latitude}, {local.longitude})"))
            page.update()

        page.apennd(
            {local.latitude},
            {local.longitude},
            {local.adress},
        )

app(target=main, view=AppView.WEB_BROWSER,assets_dir='assets')

Verify issue the method ;  Lok = geolocator.is_location_service_enabled()
 
How do i solve this problem?

**Comment by celsoabreu** [comment]
I would like support contact to provide feedback and returns in a time manner, please!

**Reply by FeodorFitsner** [reply]
Could you please elaborate what the problem is?

**Comment by celsoabreu** [comment]
Tanks, return!


   1. I am having problems using Geolocator, when I try to test the is
   _location_service_enabled() method the code gives an error and does not
   recognize the assignment and the method. Could anyone help with the
   syntax?


Exxecure and loock line   Lok = geolocator.is_location_service_enabled()

import flet as ft
from flet import *

async def main(page: Page):
    #page.window.always_on_top = True
    page.on_error = lambda e: print(f"Page Error: {e.data}")
    page.scroll = ScrollMode.ADAPTIVE
    page.appbar = AppBar(title=ft.Text("Geolocator Tests"))
    geolocator = Geolocator()
    page.overlay.append(geolocator)
    Lok = geolocator.is_location_service_enabled()
    print(geolocator, geolocator, type(geolocator))

    page.add(Text(value="Hello, world!"),
         Text(f"Geolocator: {geolocator}"),
         Text(f"Type: {type(geolocator)}"),
    )

    #page.add(Text('Gelocalização do usuario', weight='bold',
title_size=15,))

    if Lok == False:
        page.apennd(msg = Text("Voce precisa ativar a localização do seu
dispositivo location/app settings. "))
    else:
        local = geolocator.get_current_position()
        print(local, {local.latitude}, {local.longitude})

        if local == False:
            page.apennd(msg = Text("Voce precisa ativar a localização do
seu dispositivo location/app settings. "))
            page.update()
        else:
            page.apennd(Text(f"get_current_position: ({local.latitude},
{local.longitude})"))
            page.update()

        page.apennd(
            {local.latitude},
            {local.longitude},
            {local.adress},
        )


app(target=main, view=AppView.WEB_BROWSER,assets_dir='assets')

Em qui., 27 de jun. de 2024 às 21:03, Feodor Fitsner <
***@***.***> escreveu:

> Could you please elaborate what the problem is?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/flet-dev/flet/discussions/3562#discussioncomment-9899356>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACZVMN6HXVLS44MGOFKVTZ3ZJSR63AVCNFSM6AAAAABKA26AFCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQOJZGM2TM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Celso Abreu
Cel.: +55 71 99245-7590 Whatsapp
e-mail: ***@***.***

**Comment by celsoabreu** [comment]
Please outher problem!
 7
Texfiel MASK

How insert mask input texfield, exemple:

Celular : +55 071 992457590 format  @+99 999 999999999
CPF: 999.999.999-99
CNPJ: 999.999/9999-99
DATE: DD/MM/YYYY - format 27/06/2024

Tanks, I hope it helps to improve this fantastic framework !






Em qui., 27 de jun. de 2024 às 21:03, Feodor Fitsner <
***@***.***> escreveu:

> Could you please elaborate what the problem is?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/flet-dev/flet/discussions/3562#discussioncomment-9899356>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ACZVMN6HXVLS44MGOFKVTZ3ZJSR63AVCNFSM6AAAAABKA26AFCVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TQOJZGM2TM>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Celso Abreu
Cel.: +55 71 99245-7590 Whatsapp
e-mail: ***@***.***

**Comment by celsoabreu** [comment]
Hello, Feador, how are you?
I'm getting in touch to see if you're interested in me being an official
promoter of Flet here in Brazil.
 You could try to open a contact with the government school and technology
innovation through a state or federal state body to promote the
dissemination of this wonderful framework as Python is already used in many
colleges here in Brazil.  yesterday I had the pleasure of participating in
a selective event where there were several technology professionals and
students and the highlights were Python and golang.  the highlight of
golang was the tool's performance as it had a workload with better
performance than other technologies.  So I would like to hear from you
about How does Flet behave if we were to do a workload test on a WMS or a
Google cloud?


-- 
https://www.linkedin.com/in/celso-abreu-b56a0225
